scheduler/src/cook/mesos/schema.clj

```diff
@@ -673,7 +678,9 @@
                    :params [db job-ent]
                    :code
                    (let [done-statuses #{:instance.status/success :instance.status/failed}
-                         mea-culpa-limit (or (:scheduler.config/mea-culpa-failure-limit
+                         mea-culpa-limit (or (when (:job/disable-mea-culpa-retries job-ent)
```

Comment:
It's better for the first case (disable-mea-culpa-retries) but worse for the second (mea-culpa-failure-limit) so to me, it's a toss-up. I picked or because (presumably) you'd want to introduce a let for mea-culpa-failure-limit.